Support raw tracepoint program type (#2542)
iovisor/bpftrace: man/adoc/bpftrace.adoc

.Additional information
* https://www.kernel.org/doc/html/latest/trace/tracepoints.html

> .variants

Reviewer: ```suggestion
[#probes-rawtracepoint]
=== rawtracepoint

.variants
```

* `rawtracepoint:event`

.shortnames
* `rt`

The hook point triggered by `tracepoint` and `rawtracepoint` is the same.
`tracepoint` and `rawtracepoint` are nearly identical in terms of functionality. The only difference is in the program context. `rawtracepoint` offers raw arguments to the tracepoint while `tracepoint` applies further processing to the raw arguments. The additional processing is defined inside the kernel.

> Tracepoint arguments are available in the `argN` which can be found in the relative path of the kernel source code `include/trace/events/` for more details.

Reviewer: ```suggestion
Tracepoint arguments are available via the `argN` builtins. The available arguments can be found in the relative path of the kernel source code `include/trace/events/`.
```
